bindings/opencv/dnn/readme.md

```diff
@@ -1,6 +1,6 @@
 # DNN example
 
 ### Overview
-This example demonstrates object detection on the depth frame using the [SSD Mobilenet object detection example from OpenCV](https://github.com/opencv/opencv/blob/3.4.0/samples/dnn/ssd_mobilenet_object_detection.cpp) and the Aditof SDK.
+This example demonstrates object detection on the a combination between depth and ir frames using the [SSD Mobilenet object detection example from OpenCV](https://github.com/opencv/opencv/blob/3.4.0/samples/dnn/ssd_mobilenet_object_detection.cpp) and the Aditof SDK.
```

Comment:
I think 'a' or 'the' is extra here because the hole sentence doesn't sound right.

Reply:
modified in commit: 12ef2f1c34a3c0a54f11b489560750b1f119899f